### PR TITLE
add sig-cloudprovider-apiserver-network-proxy job and dashboard

### PR DESCRIPTION
**Pull request by Sh4d1**
Signed-off-by: Patrik Cyvoct <patrik@ptrk.io>

In order to set annotations on jobs #14973  cc @cblecker

### CODE REVIEW

File: config/testgrids/kubernetes/sig-cloud-provider/config.yaml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+dashboard_groups:
+- name: sig-cloudprovider
+  dashboard_names:
+    - sig-cloudprovider-apiserver-network-proxy
```
Comment (justaugustus) [inline config/testgrids/kubernetes/sig-cloud-provider/config.yaml]:
```suggestion
    - sig-cloud-provider-apiserver-network-proxy
```

File: config/testgrids/kubernetes/sig-cloud-provider/config.yaml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+dashboard_groups:
+- name: sig-cloudprovider
```
Comment (justaugustus) [inline config/testgrids/kubernetes/sig-cloud-provider/config.yaml]:
```suggestion
- name: sig-cloud-provider
```

File: config/testgrids/kubernetes/sig-cloud-provider/config.yaml [hunk 3]
```diff
@@ -0,0 +1,7 @@
...
+  dashboard_names:
+    - sig-cloudprovider-apiserver-network-proxy
+
+dashboards:
+- name: sig-cloudprovider-apiserver-network-proxy
```
Comment (justaugustus) [inline config/testgrids/kubernetes/sig-cloud-provider/config.yaml]:
```suggestion
- name: sig-cloud-provider-apiserver-network-proxy
```

**Comment by Sh4d1** [comment]
@justaugustus fixed :)

**Comment by justaugustus** [comment]
Thanks @Sh4d1!
/lgtm
/approve
/assign @cblecker @krzyzacy

**Pull request merged**